docs/operations-guide/environment-variables.md

```diff
@@ -1,6 +1,6 @@
 # Environment variables
 
-Many settings in Metabase can be viewed and modified in the Admin Panel, or set via environment variables. The environment variables always take precedence.
+Many settings in Metabase can be viewed and modified in the Admin Panel, or set via environment variables. The environment variables always take precedence if you run a version 40+. Note that the environment variables won't get written into the application database.
```

Comment:
Our documentation is versioned, so this has not been a problem for about 8 months, so I would probably not include such comment anymore (and if you did, then the version numbers we use are 0.40.0).
For reference #16644
```suggestion
Many settings in Metabase can be viewed and modified in the Admin Panel, or set via environment variables. The environment variables always take precedence. Note that the environment variables won't get written into the application database.
```

Reply:
I would really want this knowledge to be persisted somewhere. There are customers that don't want or can't upgrade to newer versions and they'll face this at some point

Reply:
Okay, then fix it in the older versions of the documentation. It was also only a problem, when there where a key in the table `setting` - the envvar would still take precedence otherwise.
Again, if we had to include every bug into our documentation, then it would become unmanageable, specially when this was already fixed 8 months ago.